lib/Mojo/WebSocketProxy/Dispatcher/Config.pm

```diff
@@ -0,0 +1,59 @@
+package Mojo::WebSocketProxy::Dispatcher::Config;
+
+use Mojo::Base -base;
+
+my $actions;
+my $config;
+
```

Comment:
If you create two config instances (e.g. for test or for different proxing), it will fail and would lead to hard-to-find bugs,